packages/server/src/index.ts

```diff
@@ -133,6 +133,16 @@ io.on("connection", (socket) => {
       await socket.leave(topic);
     }
   });
+
+  socket.on("select_seat", async ([game_id, seat, nr_of_players]) => {
```

Comment:
Why is the client responsible for telling us `nr_of_players`?  Shouldn't the server know this?

Reply:
This is to avoid that the server needs to fetch the game from the data base. Although now that I think about it, without server side validation this could be abused to avoid the unsubscription.

Reply:
Ah yeah, I'm not so concerned about abuse (clients still have access to generic `subscribe`), just that there is some additional complexity introduced to the client

Reply:
Should I make it so the server fetches the game object from db and gets the number of players that way?

Reply:
Ohh, I think a db hit is a fine reason to do it the way you have, I'll mark this resolved!